s2graphql/src/main/resources/application.conf

```diff
@@ -0,0 +1,18 @@
+# DB
+#db.default.driver="com.mysql.jdbc.Driver"
+#db.default.url = "jdbc:mysql://"${host}":3306/graph_dev?useSSL=false"
+#db.default.user = "graph"
+#db.default.password = "graph"
+
+db.default.driver = "org.h2.Driver"
+db.default.url = "jdbc:h2:file:/Users/daewon/database_meta/s2graph_meta_store;MODE=MYSQL"
```

Comment:
@daewon is there any reason to use absolute path for this?

Reply:
I'm sorry
It is my mistake. The line must be cleared.